### PR TITLE
Add a numeric TTL Countdown if the time left is under 1 minute

### LINKED ISSUE #901: TTL Display Clock Alternative
whereas the ticker clock is space-savvy, it doesn't give any precise information about how much time is left to live for each message (even not when hovering over it). Would certainly increase productivity of simply displaying numeric value of TTL countdown.

### LINKED ISSUE #1020: TTL Display Clock Alternative
whereas the ticker clock is space-savvy, it doesn't give any precise information about how much time is left to live for each message (even not when hovering over it). Would certainly increase productivity of simply displaying numeric value of TTL countdown.

### PR DESCRIPTION
**Pull request by konstantinullrich**
### Contributor checklist:

* [x] My commits are in nice logical chunks with [good commit messages](http://chris.beams.io/posts/git-commit/)
* [x] My changes are [rebased](https://medium.freecodecamp.org/git-rebase-and-the-golden-rule-explained-70715eccc372) on the latest [`clearnet`](https://github.com/loki-project/loki-messenger/tree/development) branch
* [x] A `yarn ready` run passes successfully ([more about tests here](https://github.com/loki-project/loki-messenger/blob/master/CONTRIBUTING.md#tests))
* [x] My changes are ready to be shipped to users

### Description
Closes oxen-io/session-desktop-temp#911 
Add a numeric TTL Countdown if the time left is under 1 minute because it could increase productivity.

### CODE REVIEW

**Comment by Mikunj** [comment]
Could you post a screenshot/gif of how this looks in the application?

**Comment by konstantinullrich** [comment]
image

**Review by Mikunj — APPROVED** [review]
I'll also wait for approval from @Bilb before merging

**Review by Bilb — APPROVED** [review]

**Pull request merged**
